src/test/resources/liquibase/harness/change/changelogs/databricks/alterTableProperties.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same as in https://github.com/liquibase/liquibase-databricks/pull/158 -add json and yaml for reference